src/ess/reflectometry/types.py

```diff
@@ -126,3 +101,23 @@ class SampleSize(sciline.Scope[Run, sc.Variable], sc.Variable):
 Gravity = NewType('Gravity', sc.Variable)
 """This parameter determines if gravity is taken into account
 when computing the scattering angle and momentum transfer."""
+
+
+class FullData(sciline.Scope[Run, sc.DataArray], sc.DataArray):
+    pass
+
+
+class MaskedFullData(sciline.Scope[Run, sc.DataArray], sc.DataArray):
+    pass
```

Comment:
What is "full"?

Reply:
Full here means: has added coordinates. But I agree it's a poor name.